### DISCUSSION TITLE
Python Support for Prompty

### DISCUSSION THREAD

**Discussion by awonglk** [microsoft/semantic-kernel#8176] [General]
Does anyone know if the Python version of Semantic Kernel supports the use of Prompty generated prompt templates? 
It appears to be supported on CSharp only, and I'm wondering if this will come to Python soon

**Comment by moonbox3** [comment]
Hi @awonglk, we're tracking this work in the following GitHub issue in our Python backlog: https://github.com/orgs/microsoft/projects/866/views/59?sliceBy%5Bvalue%5D=python&pane=issue&itemId=67746576
